### ISSUE TITLE
graph-your-inbox update?

### ISSUE THREAD

**Issue by davidszedely** [mrenvoize/graph-your-inbox#1]
Dear Martin, any chance it would be possible to update graph-your-inbox to comply with Manifest 2 / work with new Google Chrome? Seems like an amazing tool with nearly no alternatives... Best regards, David

**Comment by mrenvoize** [comment]
Indeed, it was a fantastic tool back in the day, but alas i've never had the time to get to grips with the codebase and bring it up to date.  I mostly grabbed a copy of the code, cloned it up here to work on, then paid work took over my time again.

**Comment by joshdance** [comment]
So just to clarify, this is not working right now? Wondering if I can use it on my local machine etc.

**Comment by mrenvoize** [comment]
Indeed.. it's not working right now.. would love to see someone fork it and get it working again as I loved it.. but I don't have time to do it myself at the moment :(
